<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefaultAnnoDialog</class>
 <widget class="QDialog" name="DefaultAnnoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>图片文件夹：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>项目名称:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>label(逗号;分隔):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="labels_textEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="open_dir_Button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="project_textEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>标注文件保存地址：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="fileDir_textEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="save_anno_textEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="annoType_comboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>分类_classifiy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>点_points</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>矩形框_rect</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>分割_segement</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3D_目标检测</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>标注类型:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
